draft-ietf-tls-tls13.md

```diff
@@ -1,4 +1,4 @@
----
+/---
```

Comment:
I can�t figure out which change this applies to.

spt

On May 16, 2015, at 17:11, ekr <notifications@github.com> wrote:

> In draft-ietf-tls-tls13.md:
> 
> > @@ -1,4 +1,4 @@
> > ----
> > +/---
> 
> This seems like it might be wrong.
> 
> �
> Reply to this email directly or view it on GitHub.
>